webcompat/templates/contributors.html

```diff
@@ -143,6 +143,26 @@ <h4 class="headline-3">Romania</h4>
         </div>
       </article>
 
+      <!-- Swarnava Sengupta -->
```

Comment:
Please add your name per alphabetic order as the list is organized like this. thanks!